docs/Change_Log.md

```diff
@@ -1,10 +1,33 @@
 # Change Log
 
+## Changes to Bot Permissions for Interactions and Webhooks
+
+> danger
+> This entry includes breaking changes
+
+#### Jun 29, 2022
+
+#### Upcoming Changes
+
+Starting **July 28, 2022**, the way a bot's [permissions](#DOCS_TOPICS_PERMISSIONS/permissions) are calculated is changing in two cases:
+- When **responding to an [interaction](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING)** (like application commands or message components)
+- When **executing a [webhook](#DOCS_RESOURCES_WEBHOOK) that the bot created**
+
+Going forward, in the above cases, a bot’s permission will be calculated based on the permissions it was installed with and any overrides in the associated channel. Previously, a bot’s permissions matched `@everyone`, regardless of the permissions it was installed with.
```

Comment:
This is confusing, "installed with" suggests that moderators cannot change the permission in the role after adding the both, which is not true, unless this is truly relying on the granted permissions from the oauth step.

It also suggests that its not a combined check like the rest of discords permission system, wherein if `@everyone`  has permission, so would the bot (unless explicitly denied in an overwrite)